kubernetes/provider.go

```diff
@@ -148,36 +148,35 @@ func Provider() terraform.ResourceProvider {
-			"kubernetes_role":                             resourceKubernetesRole(),
-			"kubernetes_secret":                           resourceKubernetesSecret(),
-			"kubernetes_service":                          resourceKubernetesService(),
-			"kubernetes_service_account":                  resourceKubernetesServiceAccount(),
-			"kubernetes_stateful_set":                     resourceKubernetesStatefulSet(),
-			"kubernetes_storage_class":                    resourceKubernetesStorageClass(),
-			"kubernetes_validating_webhook_configuration": resourceKubernetesValidatingWebhookConfiguration(),
-			"kubernetes_mutating_webhook_configuration":   resourceKubernetesMutatingWebhookConfiguration(),
+			"kubernetes_api_service":               resourceKubernetesAPIService(),
+			"kubernetes_cluster_role":              resourceKubernetesClusterRole(),
+			"kubernetes_cluster_role_binding":      resourceKubernetesClusterRoleBinding(),
+			"kubernetes_config_map":                resourceKubernetesConfigMap(),
+			"kubernetes_cron_job":                  resourceKubernetesCronJob(),
+			"kubernetes_daemonset":                 resourceKubernetesDaemonSet(),
+			"kubernetes_deployment":                resourceKubernetesDeployment(),
+			"kubernetes_endpoints":                 resourceKubernetesEndpoints(),
+			"kubernetes_horizontal_pod_autoscaler": resourceKubernetesHorizontalPodAutoscaler(),
+			"kubernetes_ingress":                   resourceKubernetesIngress(),
+			"kubernetes_job":                       resourceKubernetesJob(),
+			"kubernetes_limit_range":               resourceKubernetesLimitRange(),
+			"kubernetes_namespace":                 resourceKubernetesNamespace(),
+			"kubernetes_network_policy":            resourceKubernetesNetworkPolicy(),
+			"kubernetes_persistent_volume":         resourceKubernetesPersistentVolume(),
+			"kubernetes_persistent_volume_claim":   resourceKubernetesPersistentVolumeClaim(),
+			"kubernetes_pod":                       resourceKubernetesPod(),
+			"kubernetes_pod_disruption_budget":     resourceKubernetesPodDisruptionBudget(),
+			"kubernetes_pod_security_policy":       resourceKubernetesPodSecurityPolicy(),
+			"kubernetes_priority_class":            resourceKubernetesPriorityClass(),
+			"kubernetes_replication_controller":    resourceKubernetesReplicationController(),
+			"kubernetes_role_binding":              resourceKubernetesRoleBinding(),
+			"kubernetes_resource_quota":            resourceKubernetesResourceQuota(),
+			"kubernetes_role":                      resourceKubernetesRole(),
+			"kubernetes_secret":                    resourceKubernetesSecret(),
+			"kubernetes_service":                   resourceKubernetesService(),
+			"kubernetes_service_account":           resourceKubernetesServiceAccount(),
+			"kubernetes_stateful_set":              resourceKubernetesStatefulSet(),
+			"kubernetes_storage_class":             resourceKubernetesStorageClass(),
```

Comment:
`kubernetes_validating_webhook_configuration` and `kubernetes_mutating_webhook_configuration` have gone missing here!

Reply:
Fixed